Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -42,3 +42,17 @@ public struct PlotLegend {
     public var textSize: Float = 10
     public init() {}
 }
+
+public protocol Annotation {
+    var color: Color { get set }
+    var size: Float { get set }
+    var location: Point {get set }
+}
+
+struct TextAnnotation : Annotation {
+    public var text = ""
+    public var color = Color.black
+    public var size: Float = 15
+    public var location = Point(0.0, 0.0)
+    public init() {}
+}
```

Comment:
Explicitly defining an init tells Swift not to synthesise one automatically for the stored properties. An empty init makes it so the only way to create a text annotation is to do `TextAnnotation()`. It will not be possible to specify any property in the initialiser and every `TextAnnotation` created will always have the initial values specified here. If you remove `public init() {}` Swift will synthesise two initialisers automatically: `TextAnnotation()` (this one is synthesised since there are default values for all the properties) and `TextAnnotation(text: String, color: Color, size: Float, location: Point)`.

Reply:
Ok, I will remove that now. Thank you for your review, appreciate it a lot, especially since I'm getting accustomed to Swift syntax :)

Reply:
@odmir @WilliamHYZhang although not specifying an init will generate one automatically, its access level will be internal. You'll need to specify an init like so to be able to initialize annotations outside PlotStyleHelpers.swift:

```swift
public init(text: String = "", color: Color = .black, size: Float = 15, location: Point = Point(0.0, 0.0)) {
      self.text = text
      self.color = color
      self.size = size
      self.location = location
    }
```